packages/cashier/src/stores/__tests__/account-transfer-store.spec.tsx

```diff
@@ -85,7 +85,7 @@ beforeEach(() => {
             }),
         },
         balanceAll: jest.fn().mockResolvedValue({ balance_response: { balance: '20' } }),
-        mt5LoginList: jest.fn().mockResolvedValue(),
+        mt5LoginList: jest.fn().mockResolvedValue({}),
         tradingPlatformAccountsList: jest.fn().mockResolvedValue({
```

Comment:
I believe we should create or import from component's files all related types, I mean for variables and props used in tests

Reply:
We need some mock data just like the WS response for the test and as this is a test we need to define them here.

Reply:
sure, but as I see there are no types for all mocked objects and variables. Correct me please...

Reply:
Sorted.